src/module.ts

```diff
@@ -150,14 +152,43 @@ export default defineNuxtModule<ModuleOptions>({
     // Expose resolved tailwind config as an alias
     // https://tailwindcss.com/docs/configuration/#referencing-in-javascript
     if (moduleOptions.exposeConfig) {
+      const exposeMap = {}
+      const populateMap = (obj, path = [], level = 0, maxLevel = moduleOptions.exposeLevel) => {
+        Object.entries(obj).forEach(([key, value]) => {
+          if (
+            level >= maxLevel ||
+            typeof value !== 'object' ||
+            Array.isArray(value) ||
+            Object.keys(value).find(k => !k.match(/^[0-9a-z]+$/i))
+            // || !Object.values(value).find((v) => typeof v === 'object')
```

Comment:
What about this line?

Reply:
This line is commented out to determine if an export would be an object or a simple primitive value to cut down the number of files, but it may not be reliable.

Reply:
Also checks to see if the object has any nested objects as values to create a directory for it!

Reply:
Then let's remove the commented line?